webhooks-extension/test/install_dashboard_and_extension.sh

```diff
@@ -16,7 +16,36 @@ export test_dir="${tekton_repo_dir}/webhooks-extension/test"
 source ${test_dir}/config.sh
 source ${test_dir}/util.sh
 
-echo "Installing Dashoard and webhooks extension into $DASHBOARD_INSTALL_NS namespace"
-install_webhooks_extension $DASHBOARD_INSTALL_NS
-install_dashboard $DASHBOARD_INSTALL_NS
+echo -e "\n\nPlease choose which versions you will like to install:"
+echo "1) Current Releases"
+echo "2) Nightly Builds"
+echo "3) Developments"
+
+condition_check='[[ $VERSION == "" ]] || [[ $VERSION != [1-3] ]]'
+while $(eval "${condition_check}");
+do
+    read -p "Version: " -n 1 -r VERSION
+    if $(eval "${condition_check}"); then
+      echo -e "\n\nInvalid input, try again."
+    fi
+done
+
+if [[ $VERSION == "1" ]]; then
+  VERSION_TEXT="CURRENT RELEASES"
+elif [[ $VERSION == "2" ]]; then
+  VERSION_TEXT="NIGHTLY BUILDS"
+else
+  VERSION_TEXT="DEVELOPMENT VERSIONS"
+fi
+
+echo
+# Check that GOPATH and KO_DOCKER_REPO are set properly
+check GOPATH
+check KO_DOCKER_REPO "export KO_DOCKER_REPO=docker.io/${DOCKERHUB_USERNAME}"
```

Comment:
Can you explain/comment/refactor this line please? Confusing on reading, could just be missing a newline

Reply:
what's confusing here? here it is just checking that the user has correctly specified `GOPATH` and `KO_DOCKER_REPO`

Reply:
Should it not be

check KO_DOCKER_REPO
"export KO_DOCKER_REPO=docker.io/${DOCKERHUB_USERNAME}"

?

Reply:
i see the confusion now. so the `check` method takes as arguments the value you want to check and optionally, an example of what that value looks like.